src/contributors/gilgold.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Gil Goldstein",
+    "desc": "Software Engineer at Indeed | Creator of Tabox for Chrome",
+    "github": "gilgold",
+    "twitter": "gilgoldstein",
+    "linkedin": "gilgoldstein"
```

```suggestion
    "github": "https://github.com/gilgold",
    "twitter": "https://twitter.com/gilgoldstein",
    "linkedin": "https://www.linkedin.com/in/gilgoldstein"
```
Need to update this PR by adding complete links.